spec/requests/v1/pacbio/plates_spec.rb

```diff
@@ -85,110 +103,15 @@
       end
 
       it 'does not create a plate' do
-        expect { post v1_pacbio_plates_path, params: body, headers: json_api_headers }.to_not change(Pacbio::Plate, :count)
+        expect { post v1_pacbio_plates_path, params: body, headers: json_api_headers }.to_not change(Plate, :count)
       end
 
       it 'has an error message' do
         post v1_pacbio_plates_path, params: body, headers: json_api_headers
         json = ActiveSupport::JSON.decode(response.body)
-        errors = json['data']['errors']
-        expect(errors['run']).to be_present
-      end
-
-    end
-  end
-
-  context '#update' do
-    let(:plate) { create(:pacbio_plate) }
-    let(:new_run) { create(:pacbio_run) }
-
-    context 'on success' do
-      let(:body) do
-        {
-          data: {
-            type: "plates",
-            id: plate.id,
-            attributes: {
-              pacbio_run_id: new_run.id
-            }
-          }
-        }.to_json
-      end
-
-      it 'has a ok status' do
-        patch v1_pacbio_plate_path(plate), params: body, headers: json_api_headers
-        expect(response).to have_http_status(:ok)
-      end
-
-      it 'updates a plate' do
-        patch v1_pacbio_plate_path(plate), params: body, headers: json_api_headers
-        plate.reload
-        expect(plate.run).to eq new_run
+        expect(json['data']['errors']).to be_present
```

Comment:
Make sure they are the correct errors, not just that they exist

Reply:
> Make sure they are the correct errors, not just that they exist

I already do that in the plate creator model.